src/components/tutorial-meta/tutorial-meta.module.css

```diff
@@ -12,6 +12,7 @@
 
 .heading {
 	composes: g-offset-scroll-margin-top from global;
+	margin-top: 0;
```

Comment:
This fixes an issue where `margin-top` on the `h1` was causing alignment issues.

| Before | After |
| - | - |
| ![tutorial-meta-before](https://user-images.githubusercontent.com/4624598/223175292-971deb97-d15e-4ff9-a9d0-da81310c454e.png) | ![tutorial-meta-after](https://user-images.githubusercontent.com/4624598/223175291-a62ac164-6ee5-4f67-83ab-ba4bcb8ad798.png) |

Reply:
Nice catch!